src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -35,18 +37,25 @@ public LogicManager(Model model, Storage storage) {
         this.model = model;
         this.storage = storage;
         addressBookParser = new AddressBookParser();
+        taskPanelParser = new TaskPanelParser();
     }
 
     @Override
     public CommandResult execute(String commandText) throws CommandException, ParseException {
         logger.info("----------------[USER COMMAND][" + commandText + "]");
 
         CommandResult commandResult;
-        Command command = addressBookParser.parseCommand(commandText);
+        Command command;
+        if (commandText.startsWith("task") && Character.isWhitespace(commandText.charAt(4))) {
```

Comment:
Would using TaskCommand.COMMAND_WORD here be better?

Reply:
Yup, let me update that!